docs/_plugins/nav-tabs.rb

```diff
@@ -0,0 +1,54 @@
+require "erb"
+require "securerandom"
+
+module Jekyll
+  module NavTabs
+    class NavTabsBlock < Liquid::Block
+      def render(context)
+        environment = context.environments.first
+        environment['navtabs'] = {} # reset each time
+        super
+
+        uuid = SecureRandom.uuid
+        template = ERB.new <<-EOF
+<ul class="nav nav-tabs" id="<%= uuid %>" role="tablist">
```

Comment:
```suggestion
<ul class="nav nav-tabs nav-tab-margin" id="<%= uuid %>" role="tablist">
```

https://github.com/Alluxio/alluxio/blob/8ed3925d0740eb73051a050b9bf1c778f575580d/docs/css/main.css

```css
nav-tab-margin {
  margin-bottom: 1rem;
}
```

Try values between 0.2 and 1 `rem` to see what looks reasonable.

Reply:
thanks lemme try it out